wordpress-mobile/WordPress-iOS
# Remove the .showGoogle segue and replace with programmatic nav

Fixes # n/a

This PR removes the segue that sends a user to the "Continue with Google" screen. It now navigates the user programmatically.

See also: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/219

## To test:
1. Check out this branch
2. `rake dependencies`
3. Build and run
4. Navigate through each of these flows:

#### Flow 1, Login via Google
1. Select the Log in button
2. Select Continue with Google button
3. Select Continue on the alert message
4. Enter a valid email address for an existing WordPress.com account
5. Follow the rest of the Google login flow (password and / or 2fa)
6. Verify you are navigated to the "Continue" screen

#### Flow 2, Signup via Google
1. Select the Sign up for WordPress.com button
2. Select Continue with Google button
3. Select Continue on the alert message
4. Choose an email address that isn't connected to WordPress.com yet
5. Follow the rest of the Google sign up flow (password and / or 2fa)
6. Verify you are navigated to the New Account screen


### Do not merge until:
1. The Authenticator PR has been merged: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/219 ✔️
2. A new Authenticator release has been created ✔️ 
3. The release is published on Cocoapods trunk ✔️
4. This PR points to the new release  ✔️

PR submission checklist:
